stdlib/public/SDK/Foundation/Notification.swift

```diff
@@ -31,15 +31,15 @@ public struct Notification : ReferenceConvertible, Equatable, Hashable {
     /// An object that the poster wishes to send to observers.
     ///
     /// Typically this is the object that posted the notification.
-    public var object: AnyObject?
+    public var object: Any?
     
     /// Storage for values or objects related to this notification.
     public var userInfo: [String : Any]?
     
     /// Initialize a new `Notification`.
     ///
     /// The default value for `userInfo` is nil.
-    public init(name: Name, object: AnyObject? = nil, userInfo: [String : Any]? = nil) {
+    public init(name: Name, object: Any? = nil, userInfo: [String : Any]? = nil) {
```

Comment:
@parkera Yeah, I noticed that as I was migrating. We could probably do away with that special handling now that the runtime can bridge anything for you.